extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/campaign/feedbackevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Delivery Feedback Events",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Contains all delivery event fields.",
+  "definitions": {
+    "feedbackevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/feedbackType": {
+          "title": "Feedback Type",
+          "type": "string",
+          "description": "The type of message event."
+        },
+        "https://ns.adobe.com/experience/campaign/retryCount": {
+          "title": "Retry Count",
+          "type": "integer",
+          "description": "Number of failed attempts before the current event."
+        },
+        "https://ns.adobe.com/experience/campaign/delivery": {
+          "title": "The campaign activity originating a message to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Delivery ID",
+              "type": "string",
+              "description": "The campaign activity originating this message."
+            },
+            "xdm:campaignId": {
+              "title": "Campaign Id",
+              "type": "string",
+              "description": "Campaign of which this delivery was a part."
+            },
+            "xdm:from": {
+              "title": "From",
+              "type": "string",
+              "description": "Address used as a from/sender/caller address. Depending on the communication channel, the `address` has a different format.\n\n  * `email`: an email address.\n  * `textMessage` or `phone`: a shortcode or phone number."
+            },
+            "xdm:type": {
+              "title": "Delivery Type",
+              "type": "string",
+              "description": "Mode of delivery for the messages sent."
+            },
+            "xdm:label": {
+              "title": "Delivery Label",
+              "type": "string",
+              "description": "A human-friendly name of the campaign activity which is originating this message."
+            },
+            "xdm:name": {
+              "title": "Delivery Internal Name",
+              "type": "string",
+              "description": "A human-friendly identifier of the campaign activity which is originating this message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/message": {
```

Comment:
Maybe instead of the "message", say it "email/message", then you can use xdm: subject and xdm: size more as properties as they semantically make the properties much richer.

Reply:
Same comment as above. The message block would common for other channels as well.